disk_objectstore/container.py

```diff
@@ -1393,6 +1395,17 @@ def pack_all_loose(  # pylint: disable=too-many-locals,too-many-branches,too-man
         # on Mac and on Windows (see issues #37 and #43). Therefore, I do NOT delete them,
         # and deletion is deferred to a manual clean-up operation.
 
+        if callback:
+            callback(
+                action="init",
+                value={
+                    "total": self.get_total_size()["total_size_loose"],
+                    "description": "Packing loose objects",
+                },
+            )
+            # I wish this would show as MB, GB. In tqdm it's easy:
+            # just pass unit='B' and unit_scale=1024. But would callback accept **keywords?
```

Comment:
https://github.com/aiidateam/aiida-core/blob/cc62561ccb8c4a8f0f85f81ae08ce8ae8f8dbec0/src/aiida/cmdline/commands/cmd_storage.py#L191

Reply:
Should be even easier, since it is a partial function so one should be able to pass it to `progress_reporter`. You might need a check if the the `progress_reporter` is  actually tqdm

Reply:
Thanks @agoscinski , this will be a bit tricky, because in that case we have to change the total value of the bar from _items_ to _byte_. 
And some of the thing has to be done in `aiida-core`, I feel like this makes `disk-objectstore` less and less transferable.

Reply:
I think this can be done fully in aiida-core in the create_callback function, but I can give it a go after this PR, if the progress bar is really not so readable.

Reply:
Very good, suggestion @agoscinski . However that requires some change in `aiida-core` I opened an issue for future references : https://github.com/aiidateam/aiida-core/issues/6564

I think for now, we can accept the way it is.